spec/models/review_spec.rb

```diff
@@ -5,4 +5,14 @@
   it { should belong_to(:creator) }
   it { should validate_presence_of(:description) }
   it { should validate_presence_of(:rating) }
+
+  describe ".most_recent_review" do
+    it "returns most recent review" do
+      user = Fabricate(:user)
+      video = Fabricate(:video)
+      review_old = Fabricate(:review, creator: user, video: video, created_at: 2.weeks.ago)
+      review_new = Fabricate(:review, creator: user, video: video)
```

Comment:
You should set the `created_at:` here as well, to make it explicit that this is really newer review.